### DISCUSSION TITLE
Problem with the timezone configuration.

### DISCUSSION THREAD

**Discussion by ChatNoir-040147** [ScootKit/CustomDCBot#70] [Questions] [answered]
Hi there !

I've this retunr when I start the bot (new and clean setup) : 

```
[INFO] Checking configurations...
[ERROR] Unknown type: timezone
[ERROR] Unknown type: timezone
```

but here is what i have in my configuration about the Timezone :

`  "timezone": "Europe/Berlin",`

Has anyone ever encountered this problem?

thank you in advance for your feedback

**Comment by jateute** [comment]
I hadn‘t encountered this problem, but I‘ve seen that the type `timezone` doesn’t exists in the function `checkType`. @SCDerox I think you forgot to push the file `src/functions/config.js`.

**Reply by SCDerox** [reply]
Yeah, thanks for reminding me, going to publish a fix today ^^

**Comment by SCDerox** [comment, marked as answer]
@ChatNoir-040147, should be fixed in [`e9070e0`](https://github.com/SCNetwork/CustomDCBot/commit/e9070e0ed96787e82e256becbe5edb7b16b9d4dc). Please update your bot to the latest main branch.
